packages/client/manifest.json

```diff
@@ -6723,7 +6723,21 @@
     "illegalChildren": ["section", "sidepanel"],
     "showEmptyState": false,
     "draggable": false,
-    "info": "Side panels are hidden by default. They will only be revealed when triggered by the 'Open Side Panel' action."
+    "info": "Side panels are hidden by default. They will only be revealed when triggered by the 'Open Side Panel' action.",
+    "sendEvents": true,
```

Comment:
Just confirming this is intentional. `sendEvents` will mean that we will send posthost events whenever this setting is changed by any user. We typically use it to track usage of certain things to improve our default behaviour. Just letting you know that in case this wasn't intentional.